docs/diagrams/class/CommandParser.puml

```diff
@@ -0,0 +1,53 @@
+@startuml
+
+skinparam classAttributeIconSize 0
+
+class CommandHandler {
+    - scanner: Scanner
+    - taskManager: TaskManager
+    - weekView: WeekView
+    - monthView: MonthView
+    - inMonthView: boolean
+    + CommandHandler(scanner: Scanner, taskManager: TaskManager, weekView: WeekView, monthView: MonthView)
+    + handleCommand(): void
+    - handleNextCommand(): void
+    - handlePreviousCommand(): void
+    - handleUpdateCommand(parts: String[]): void
+    - handleAddCommand(parts: String[]): void
+    - handleDeleteCommand(parts: String[]): void
+    - handleMarkCommand(parts: String[]): void
+    - handlePriorityCommand(parts: String[]): void
+    - handleMonthCommand(): void
+    - handleWeekCommand(): void
+    - printHelp(): void
+    - handleQuitCommand(): void
+}
+
+class StringParser {
```

Comment:
Is the class missing the attribute `private static final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy")`?